src/main/java/mate/jdbc/service/ManufacturerServiceImpl.java

```
@@ -0,0 +1,39 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        return manufacturerDao.get(id);
```

let's throw an exception in case manufacturer isn't found

---

Done